demos/src/character_control_systems/platformer_control_systems.rs

```diff
@@ -574,7 +570,7 @@ pub fn apply_platformer_controls(
                 // When set, the `desired_forward` of the dash action "overrides" the
                 // `desired_forward` of the walk basis. Like the displacement, it gets "frozen" -
                 // allowing to easily maintain a forward direction during the dash.
-                desired_forward: if forward_from_camera.is_none() {
+                desired_forward: if !camera_contoller.map(|c| c.following()).unwrap_or(false) {
```

Comment:
Prefer reducing negations whenever feasible, since they tend to stack up. In this case, this simply means swapping the _then_ and `else` clauses.